tests/layer_tests/tensorflow_tests/test_tf_ReduceLogicalOps.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+from common.tf_layer_test_class import CommonTFLayerTest
+
+
+class TestReduceLogicalOps(CommonTFLayerTest):
+    def _prepare_input(self, inputs_info):
+        assert 'input' in inputs_info, "Test error: inputs_info must contain `input`"
+        x_shape = inputs_info['input']
+        inputs_data = {}
+        inputs_data['input'] = np.random.randint(0, 2, x_shape).astype(bool)
+        return inputs_data
+
+    def create_reduce_net(self, shape, axis, operation, keep_dims, ir_version, use_new_frontend):
+        import tensorflow as tf
+        ops_mapping = {
```

Comment:
Could you check test_tf_LogicalOps.py? It looks like tests same operations. If not - just skip this comment.

Reply:
You are right. Let me remove my part for Logical operations then.

Reply:
I just renamed your test file to `test_tf_ReduceLogicalOps.py` to be aligned with arithmetic reduce ops